frontend/src/metabase/visualizations/echarts/pie/option.ts

```diff
@@ -0,0 +1,44 @@
+import type { EChartsOption, RegisteredSeriesOption } from "echarts";
+
+import type {
+  ComputedVisualizationSettings,
+  RenderingContext,
+} from "metabase/visualizations/types";
+
+import type { PieChartModel } from "./model/types";
+
+// static, non-computed values
+const SUNBURST_SERIES_OPTIONS: RegisteredSeriesOption["sunburst"] = {
+  type: "sunburst",
+  nodeClick: false,
+  sort: undefined,
+  emphasis: { focus: "none" },
+  label: {
+    rotate: 0,
+    overflow: "none",
+    lineHeight: 50, // TODO confirm
+    fontSize: 16, // TODO confirm
+    color: "white", // TODO select color dynamically based on contrast with slice color
+  },
+  radius: ["60%", "90%"], // TODO compute this dynamically based on side length like in PieChart.jsx
+};
+
+export function getPieChartOption(
+  chartModel: PieChartModel,
+  settings: ComputedVisualizationSettings,
+  renderingContext: RenderingContext,
+): EChartsOption {
+  return {
```

Comment:
I'm intentionally leaving the options object sparse in this PR. Later PRs will focus on adding specific parts of the object, e.g. show total setting, show percentages setting, etc